.github/workflows/register-schemas.yaml

```diff
@@ -0,0 +1,59 @@
+on:
+  push:
+    branches: master
+
+name: Register schemas & update table
+
+env:
+  RENV_PATHS_ROOT: ~/.local/share/renv
+
+jobs:
+  register-schemas-dryrun:
+    runs-on: macos-10.15
+    strategy:
+      fail-fast: false
```

Comment:
why set this to `false`? don't you want to stop on first failure?

Reply:
Good catch. It will stop on first failure because it's only a single job. However, I will remove that if I stick with the GH Action since it's not needed in situations with only one job. If there were a matrix of jobs, I would not want it to fail fast because it automatically stops all other jobs in the matrix.

From the docs:
> When set to `true`, GitHub cancels all in-progress jobs if any matrix job fails. Default: `true`